Makefile

```diff
@@ -44,12 +45,31 @@ $(call add-bindata,assets,./bindata/...,bindata,assets,pkg/operator/assets/binda
 $(call build-image,cert-manager-operator,$(IMAGE_OPERATOR),./images/ci/Dockerfile,.)
 $(call build-image,cert-manager-operator-bundle,$(IMAGE_OPERATOR_BUNDLE),./bundle/bundle.Dockerfile,./bundle)
 
+# generate scorecard image targets
+.PHONY: build/custom-scorecard-tests
+build/custom-scorecard-tests:
+	go build -mod=vendor $(GO_GCFLAGS) $(GO_ASMFLAGS) -o $(BUILD_DIR)/$(@F) ./test
```

Comment:
What does the `$(@F)` do?

Reply:
It is an automatic variable which picks up `custom-scorecard-tests` automatically from the target name `build/custom-scorecard-tests`

Reply:
Nice hack!